package.json

```diff
@@ -11,11 +11,19 @@
     "test": "jest"
   },
   "jest": {
-    "setupFiles": ["./jest.js"]
-   },
+    "setupFiles": [
+      "./jest.js"
+    ]
+  },
   "dependencies": {
     "dedent": "^0.7.0",
     "js-cookie": "^2.1.0",
-    "ramda": "^0.26.1"
+    "linkstate": "^1.1.1",
```

Comment:
Preact helper package that makes writing form components a bit easier. It's small, they recently broke it out of the core library.